test-result-summary-client/src/Build/TestTable.jsx

```diff
@@ -61,7 +61,13 @@ export default class TestTable extends Component {
 
         const renderAction = (value, row, index) => {
             const { testId, testName } = value;
-            const issueUrl = `https://github.com/eclipse/openj9/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+${testName}`;
+            let issueUrl = `https://github.com/AdoptOpenJDK/openjdk-tests/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+${testName}`;
+            if (buildData) {
+                const { buildName } = buildData;
+                if (buildName.includes('j9') || buildName.includes('ibm')) {
+                    issueUrl = `https://github.com/eclipse/openj9/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+${testName}`;
+                }
+            }

```

Comment:
Could we make this a function? Similar to `gitIssue`

Reply:
possibleIssues has been changed into a function similar to gitIssue in the new commit, and tested locally. Thanks.